client/components/domains/domain-product-price/index.jsx

```diff
@@ -16,10 +16,18 @@ var PremiumPopover = require( 'components/plans/premium-popover' ),
 const domainsWithPlansOnlyTestEnabled = abtest( 'domainsWithPlansOnly' ) === 'plansOnly';
 
 var DomainProductPrice = React.createClass( {
+	propTypes: {
+		hasLoadedFromServer: React.PropTypes.bool,
+		isPremiumRequired: React.PropTypes.bool,
```

Comment:
Small nitpick, but I think `isPlanRequired` would be a better name here, since either Premium or Business satisfy this requirement.